### PR TITLE
Implement Workflow Request Handler Create Workflow Processing Utilities

### PR DESCRIPTION
**Pull request by MahtabBukhari**
## Close: #1940

image

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by MahtabBukhari** [comment]
@humansinstitute Could you please review the PR?

**Comment by elraphty** [comment]
> @humansinstitute Could you please review the PR?

@MahtabBukhari please right a test for this

**Comment by MahtabBukhari** [comment]
@elraphty Could you please review Now?

File: utils/workflow_processor.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	"fmt"
+	"github.com/google/uuid"
+)
+
+type ProcessorConfig struct {
+	RequiresProcessing bool
+	HandlerFunc        string
+	Config             json.RawMessage
+}
+
```
Comment (elraphty) [inline utils/workflow_processor.go:14]: @humansinstitute Should these return nil for now?

File: utils/workflow_processor.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+func lookupProcessingConfig(source string) error {
+	return nil
+}
+
+func processWithHandler(requestID string) (string, error) {
+	fmt.Println("Processing with default handler")
+	return fmt.Sprintf("Processed with default handler, RequestID: %s", requestID), nil
+}
```
Comment (elraphty) [inline utils/workflow_processor.go:37]: @humansinstitute Should these return nil for now?

**Comment by MahtabBukhari** [comment]
@humansinstitute Could you please merge this PR too as cypress tests pass?

**Comment by MahtabBukhari** [comment]
@humansinstitute Could you please merge this PR too

**Pull request merged**